components/OrganizationEdit/OrganizationEdit.js

```diff
@@ -240,17 +240,21 @@ class OrganizationEdit extends Component {
     })
 
     updateOrganization(organization).then(response => {
+      let success = !response.hasOwnProperty('error')
```

Comment:
random drive-by PR comments because I still get the emails.  Feel free to ignore if too nitpicky.
s/let/const ?